src/organizations/OrganizationEntity.js

```diff
@@ -40,10 +40,22 @@ export class OrganizationEntity extends Entity {
 		return this._entity && this._entity.properties && this._entity.properties.endDate;
 	}
 
+	isAfterEndDate() {
+		var nowDate = Date.now();
+		var endDate = Date.parse(this.endDate());
+		return endDate ? endDate <= nowDate : null;
+	}
+
 	startDate() {
 		return this._entity && this._entity.properties && this._entity.properties.startDate;
 	}
 
+	isBeforeStartDate() {
+		var nowDate = Date.now();
+		var startDate = Date.parse(this.startDate());
+		return startDate ? startDate > nowDate : null;
```

Comment:
I'm slightly surprised by the default value being null rather than false. But it worked that way before so :shipit: .

Reply:
Right, I was nervous about changing it in case somewhere needs to differentiate between "No it's not" and "I don't know, I can't get an end date".